course5/homework06/homework06.cc

```diff
@@ -0,0 +1,57 @@
+#include<iostream>
+#include <cassert>
+#include<cstring>
+#include "string.h"
+
+int main()
+{
+    String mystr1 = "hello";
+    std::string str1 = "hello";
```

Comment:
Instead of doubling every test, write your test as a template function, and run it once for std::String, and once for your string.